assignment.js

```diff
@@ -112,6 +145,74 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+console.log('')
+let playerAChoice = Math.random()
+let playerBChoice = Math.random()
+
+
+console.log(playerAChoice)
+//generates playerA's decsion
+ if (playerAChoice <0.333){
+     playerAChoice = "rock"
+ }else if(playerAChoice <=0.666){
+     playerAChoice = "paper"
+ }
+ else{
+     playerAChoice = "scissors"
+ }
+
+
+//generates playerB's decsion
+if (playerBChoice <0.333){
+    playerBChoice = "rock"
+}else if(playerBChoice <=0.666){
+    playerBChoice = "paper"
+}
+else{
+    playerBChoice = "scissors"
+}
+
```

Comment:
@jct117 Good job here reusing the variable `playerAChoice` and `playerBChoice` to retrieve the random number and then set it to the choice based on the value of the random number.
One suggestion for your generators for each player decision you can move that logic into a function and reuse it. Since the code is similar you can wrap in a function and reuse